### ISSUE TITLE
Recent changes to SQLite3 are causing go get issues

### ISSUE THREAD

**Issue by plattyp** [mattes/migrate#184]
- When using `go get -u github.com/mattes/migrate` I get the following error:
```bash
/Users/rtuser/go/src/github.com/mattes/migrate/driver/sqlite3/sqlite3.go:95: impossible type assertion:
	sqlite3.Error does not implement error (Error method has pointer receiver)
```

- I think the root cause is due to recent changes to this dependent library: https://github.com/mattn/go-sqlite3/commit/cf4bd560f1588d96c502b4c3407fe1a10cef4a28

**Comment by adrianduke** [comment]
I've created a PR:

https://github.com/mattes/migrate/pull/185

**Comment by plattyp** [comment]
Thanks @adrianduke, you beat me too it!

**Comment by mattes** [comment]
merged #185
